socorro/processor/cache_manager.py

```diff
@@ -240,7 +267,17 @@ def _event_generator(self, nonblocking=False):
                             flags_list = ", ".join(
                                 [str(flag) for flag in flags.from_mask(event_mask)]
                             )
-                            logger.debug("EVENT: %s: %s", event, flags_list)
+                            if event.wd > 0:
+                                dir_path = self.watches.inv[event.wd]
+                            else:
+                                dir_path = ""
```

Comment:
This makes it easier to read the EVENT data in the logs. Previously, you'd get an event.name for a directory like `""` and you had to guess at what that was. Now it shows the full path.